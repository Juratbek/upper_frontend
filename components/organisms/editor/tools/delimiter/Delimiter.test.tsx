import { render } from '@testing-library/react';

import { Delimiter } from './Delimiter.tool';

describe('Delimiter tool', () => {
  it('snapshot', () => {
    render(<Delimiter />);
    expect(document.body).toMatchSnapshot();
  });
});
