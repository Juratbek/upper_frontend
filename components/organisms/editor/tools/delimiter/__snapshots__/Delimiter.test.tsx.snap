// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Delimiter tool > snapshot 1`] = `
<body>
  <div>
    <div
      class="delimiter"
    >
      *
         
      *
         
      *
    </div>
  </div>
</body>
`;
